mne/decoding/classifier.py

```diff
@@ -269,7 +269,6 @@ def transform(self, epochs_data, y=None):
 
         _, n_freqs = psd.shape
         psd = psd.reshape(n_epochs, n_channels, n_freqs)
-
```

Comment:
why was this done. I see some extraneous lines deleted / added not related to the PR